### PR TITLE
Update to Airlift 0.178

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-example-http/pom.xml [hunk 1]
```diff
@@ -26,13 +26,6 @@
             <artifactId>json</artifactId>
         </dependency>
 
-        <dependency>
-            <!-- airlift json has a dependency on joda -->
-            <groupId>io.airlift</groupId>
-            <artifactId>joda-to-java-time-bridge</artifactId>
-            <scope>runtime</scope>
```
Comment (findepi) [inline presto-example-http/pom.xml]: Is this the only one that does not need the bridge anymore?

**Comment by findepi** [comment]
Travis:
```
[ERROR] Failures: 
[ERROR]   TestSqlStageExecution.testFinalStageInfo:93->testFinalStageInfoInternal:153 expected [true] but found [false]
```

https://github.com/prestodb/presto/issues/12095

**Comment by electrum** [comment]
Thanks, restarted the failed job.

File: presto-example-http/pom.xml (hunk 1 above)
Comment (electrum) [reply presto-example-http/pom.xml]: I checked before and all of the other plugins that use it have a direct dependency on `joda-time`. @haozhun helpfully put it directly after the `joda-time` dependency in all the POMs, so it was easy to verify using `ack -B10 joda-to-java`.

**Pull request merged**